pkg/apis/messaging/v1/channel_types.go

```diff
@@ -66,6 +66,7 @@ var (
 type ChannelSpec struct {
 	// ChannelTemplate specifies which Channel CRD to use to create the CRD Channel backing this Channel.
 	// This is immutable after creation. Normally this is set by the Channel defaulter, not directly by the user.
+	// +optional
```

Comment:
should the json tag also have omitempty?

Reply:
yeah, sounds good